lives-of-primitive-quakers/modernized/20-chapter-19.adoc

```diff
@@ -244,11 +243,11 @@ Secondly.
 a measure of the light of his own Son;–a measure of grace;--or, a measure of the Spirit;
 which the scripture expresses by several names,
 as sometimes of '`The seed of the kingdom,`' (Matt. 13:18-
-19) "`The light that makes all things manifest,`' (Eph. 5:
-13) '`The word of God,`" (Rom. 10:17) or,
-'`Manifestation of the Spirit given to profit withal,' (1 Cor. 12:
-7) "`A talent,`' (Matt. 25:15) "`A little leaven,`" (Matt.
-xiii.33;) '`The gospel preached in every creature,`" (Col. 1:23)
+19) '`The light that makes all things manifest,`' (Eph. 5:
+13) '`The word of God,`' (Rom. 10:17) or,
+'`Manifestation of the Spirit given to profit withal,`' (1 Cor. 12:
+7) '`A talent,`' (Matt. 25:15) '`A little leaven,`' (Matt.
+xiii.33;) '`The gospel preached in every creature,`' (Col. 1:23)
```

Comment:
ack, sorry, the converter/splitter really struggled here. I think this paragraph should be gone over a second time and the trailing verses should be brought up to the lines before.